File: mario-less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    // h = height, r = row, s = space, c = column
    int h;
    do
    {
    h = get_int("Height: ");
    }
    while (h < 1 || h > 8);

    {
        for(int r = 0; r < h; r++)
        {
            for(int s = 0; s < h - r - 1; s++)
            {
                printf(" ");
            }
            for(int c = 0; c <= r; c++)
            {
                printf("#");
            }
            printf("\n");
        }
    }
}

File: sentimental-mario-more/mario.py
# Import the get_int from cs50
from cs50 import get_int

# ask for height and check if user provided a valif value
while True:
    height = get_int("Specify height of the pyramide: ")
    if height > 0 and height < 9:
        break

# print pyramide
for i in range(height):
    print(" " * (height - i - 1), end="")
    print("#" * (i + 1), end=" ")
    print(" " + "#" * (i + 1),)